pkg/kube/translation/ingress.go

```diff
@@ -15,19 +15,20 @@
 package translation
 
 import (
+	"go.uber.org/zap"
```

Comment:
Please classify the import packages.

This is a 3rd party packages, and it should be put with other 3rd party packages.